LDAP-primer.md

```diff
@@ -150,16 +157,19 @@ variety of reasons, this never happened, and the directories remained disparate.
 The X.500 naming system has countries and registered organizations as DN root
 elements, which wasn't well suited to domain names used for naming on the internet,
 so a method of using domain names in DNs was standardized in RFC 2247 and is
-predominantly used in LDAP today.
+predominantly used in LDAP today. You will still see remnants of the original X.500
+naming structure especially inside X.509 certificates, but most LDAP directories still
+have organization entries (usually with an "o" RDN) and organizationalUnit entries
+(usually with an "ou" RDN).
```

Comment:
I'll leave out this aside about X.509 naming, since I don't think it's strictly necessary for understanding.